features/info_frontend.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Manuals frontend
+  Background:
+    Given I am testing through the full stack
+    And I force a varnish cache miss
+
+  @normal
+  Scenario:
+    When I visit "/info/browse/benefits"
+    Then I should see "Benefits"
+     And I should see "Unique pageviews"
+     And I should see "Searches started from this page"
+     And I should see "Problem reports"
+     And I should see "Users left by searching for"
```

Comment:
Should there be a step here where we check the meta tag that it is rendered by "info-frontend"?

I'm also not sure we need to check each of these areas exist. I'd have thought that so long as we have "Benefits" that would be sufficient - rather than needing to test that features provided by info-frontend are present.

Reply:
Could do, but none of the other tests do that at the moment, so it would be a bit inconsistent.

Reply:
Sure, If I remember right that was something we identified as an improvement in that we're testing info-frontend but we don't actually know if info-frontend is serving. 

Could be fine as a separate story though to add this to all ones that are intended to test a particular app.

Reply:
I'll make a ticket for that.